### DISCUSSION TITLE
예약 성공 감사합니다:) + 카카오 관련

### DISCUSSION THREAD

**Discussion by PREMI3R** [kimytsc/covid-rest-vaccine-macro#69] [General]
![succ](https://user-images.githubusercontent.com/89119016/129991198-6b84418e-e95d-4840-ba68-44903010bb99.png)

네이버, 카카오 동일지역으로해서 돌렸는데

돌린지 10분도 안되서 바로 잡혀버렸네요

동일병원으로 떴고 네이버쪽 성공, 카카오쪽 마감 실패로 됐네요

얼른 다녀오겠습니다 다른분들도 건투를 빌어요

+ 카카오 관련 현재 카카오 업데이트로 정상작동이 되지 않는데

혹시 응용하실 수 있는분들 팁 드리면

카카오톡 모바일에서 백신예약페이지 패킷 캡쳐 돌려보시면

통신과정 헤더중에 authorization 라고 인증헤더가 있습니다.

아마 카카오 API관련 개인계정 인증키 일텐데

authorization 헤더와 본인 인증키값을 통신시에 1회만 넣어주면

PC에서 접속시 자동으로 Redirect 시키면서 

authorization 헤더가 일치할시 kavacto라는 cookie를 생성해줍니다.

이 쿠키가 있으면 카카오 예약페이지 및 매크로 정상적으로 작동합니다 (스샷)

**Comment by kimytsc** [comment]
카카오 로그인 관련하여 설명 감사합니다!!

저도 참고하여 테스트해보고 스크립트에서 대처할 수 있을지 확인해보겠습니다.

**Reply by PREMI3R** [reply]
노고가 많으십니다 :)
방금 여자친구도 같이 예약 성공했네요
덕분에 일정 앞당겨 맞습니다 감사합니다
